service_catalog/forms/admin_forms.py

```diff
@@ -98,9 +104,55 @@ class ServiceForm(ModelForm):
                              required=False,
                              widget=forms.FileInput())
 
+    billing = forms.ChoiceField(
+        label="Billing :",
+        choices=[
+            ('defined', 'Define billing'),
+            ('User define billing', (
+                ('choice_restricted', 'Restricted billing groups'),
+                ('choice', 'All billing groups')
+            ))
+        ],
+        initial='defined',
+        widget=forms.RadioSelect(attrs={'class': 'disable_list_style'})
+    )
+
+    billing_group_id = forms.ChoiceField(
```

Comment:
The billing group ID here is never checked in a clean method.
What if the user provide a non existing ID?
What if the user provide an ID and it was supposed to be blocked?
What if the user provide an ID from a billing group it doesn't belong when the restricted choice is enabled?
All of those cases are tested?

Reply:
Django did it for non existing ID : 
![image](https://user-images.githubusercontent.com/40194291/127121900-d9cb6bf7-2a0a-4a1f-8805-356d950a5b26.png)

ID blocked by?

It will be in cutomer form. The billing_id setted by admin in this form is for the case when admin force billing so we don't care about user's billing groups.

Reply:
But test for this form must be added

Reply:
Ok, indeed if's more for the next form.
This one only need to check the given billing group ID if provided